Blackjack/Controller.cs

```diff
@@ -55,22 +57,12 @@ public void Play()
             var shuffledDeck = Shuffle();
             // deal cards - 2 cards to each player
             Deal(shuffledDeck);
+            var input = _input.ReadLine();
+            // TODO: if input is valid InputParser.ParseChoice();
+            //          else validate ?? what to return if input not valid

```

Comment:
**Background**: `return` is just one control flow mechanism. It's not always appropriate. In the context of a `void` method, `return` is the equivalent of just ending the method call. 

**Prompt**: What is it that you what to happen in terms of control flow when an invalid input is received? If you were to `return` anything here, you'd essentially end the program unless you call `Play` again at some point.